File: scratch.py
import urllib.request, json
url = "https://api.themoviedb.org/3/movie/1439930?append_to_response=videos,credits,similar&api_key=ff51f94da53ecfd4393e38a0f0112315&language=en-US"
try:
    res = urllib.request.urlopen(url)
    data = json.loads(res.read())
    print("Success, keys:", data.keys())
except Exception as e:
    print("Error:", e)

File: scratch.ts
import { config } from './src/lib/config';

async function test() {
  console.log("URL:", `${config.tmdbBaseUrl}/movie/550?api_key=${config.tmdbApiKey}&language=en-US`);
  const res = await fetch(`${config.tmdbBaseUrl}/movie/550?append_to_response=videos,credits,similar&api_key=${config.tmdbApiKey}&language=en-US`);
  console.log("Status:", res.status);
  const data = await res.json();
  console.log("Data ID:", data.id, "Title:", data.title);
}

test();
